revolv/templates/base/partials/nav.html

```diff
@@ -26,7 +26,13 @@
                <li class="user-controls has-dropdown">
                    <a href="#">{{user.username}}</a>
                    <ul class="dropdown">
+                       {% if is_administrator %}
```

Comment:
The thing about the dashboard going to the highest level was already fixed in #140, so we can just make this like go to "/dashboard/" and I believe it should be fine. @aykamko can confirm